app/src/main/java/com/example/dine_and_donate/EventActivity.java

```diff
@@ -42,20 +56,29 @@
     private Spinner mEndHour;
     private Spinner mEndMin;
     private Spinner mEndHalf;
-
     private EditText mEtEventInfo;
     private Button mBtnCreate;
+    private Button mChooseImage;
+    private ImageView ivVoucher;
+
+    private User mCurrentUser;
+
+    private Uri mSelectedImage;
+    private FirebaseUser currentUser;
```

Comment:
having both User and FirebaseUser is kinda confusing, let's maybe call this mFirebaseCurrentUser? And add a comment explaining why we need both?